packages/cosmic-swingset/lib/ag-solo/vats/vat-http.js

```diff
@@ -138,12 +141,15 @@ export function buildRootObject(vatPowers) {
     // devices.command invokes our inbound() because we passed to
     // registerInboundHandler()
     async inbound(count, rawObj) {
+      // Launder the data, since the command device tends to pass device nodes
```

Comment:
This seems weird.  This seems like a band-aid for a deeper underlying issue -- should we be having a ticket for whatever that issue is?

Reply:
It is a band-aid.  The issue (PR attempting to reproduce) is https://github.com/Agoric/agoric-sdk/pull/1956.